### PR TITLE
Footer header

### PR DESCRIPTION
**Pull request by stefanadamski**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: deploy
```
Comment (awojciak) [inline .github/workflows/deploy.yml:1]: get rid of changes in first four files

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -44,7 +44,7 @@ export const Header: React.FC<IHeader> = ({
 
   const isXsDown = useMediaQuery(theme.breakpoints.down('xs'))
 
-  const routes = ['swap', 'pool']
+  const routes = ['pool','swap']
```
Comment (awojciak) [inline src/components/Header/Header.tsx:47]: swap should be the first route

File: src/components/Header/style.ts [hunk 3]
```diff
@@ -64,12 +65,12 @@ const useStyles = makeStyles<Theme, { connected: boolean }>((theme: Theme) => ({
   }),
   link: {
     textDecoration: 'none',
-    minWidth: 'fit-content'
+    minWidth: 'fit-content',
```
Comment (awojciak) [inline src/components/Header/style.ts:68]: fix trunk issues

File: src/components/HeaderButton/style.ts [hunk 4]
```diff
@@ -19,7 +19,7 @@ const useStyles = makeStyles((theme: Theme) => ({
     },
 
     '&:hover': {
-      background: colors.invariant.componentOut3
+      background: '#4B5983'
```
Comment (awojciak) [inline src/components/HeaderButton/style.ts:22]: we haven't got this color in our design system?

**Review by awojciak — CHANGES_REQUESTED** [review]
1) check hovers in modals and fix them according to modals
2) connect wallet button isn't looking like on mockups
3) switch logo for the new one

File: src/components/HeaderButton/style.ts (hunk 4 above)
Comment (stefanadamski) [reply src/components/HeaderButton/style.ts:22]: background added to themes

Comment (stefanadamski) [reply src/components/HeaderButton/style.ts:22]: done

File: src/components/Header/style.ts (hunk 3 above)
Comment (stefanadamski) [reply src/components/Header/style.ts:68]: done

File: src/static/theme.ts [hunk 5]
```diff
@@ -21,7 +21,8 @@ export const colors = {
     neon: '#08F7FE',
     astel: '#48ADF1',
     bastille: '#1E1A23',
-    charade: '#272735' // v2.0 component
+    charade: '#272735',// v2.0 component
+    deep: '4B5983'
```
Comment (awojciak) [inline src/static/theme.ts:25]: hash is missing here

File: src/components/Header/style.ts (hunk 3 above)
Comment (awojciak) [reply src/components/Header/style.ts]: no, there is still a lot of lint errors, fix them

**Review by awojciak — COMMENTED** [review]
obraz
padding here and in wallets modal looks to be too big in comparison with mockups

**Review by awojciak — COMMENTED** [review]
obraz
it doesn't look like this on mockups

**Review by awojciak — COMMENTED** [review]
obraz
there is no color change on hover

**Review by awojciak — APPROVED** [review]

**Pull request merged**